## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 515363`
Commit: `fe44bba00ae546188446a7dabf00f8ff9ce8f385`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>alexandria</li>
  </ul>
</details>
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>cargo-tauri_1</li>
    <li>deadlock-mod-manager</li>
    <li>gitbutler</li>
    <li>kanri</li>
    <li>sftool-gui</li>
    <li>venator</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 49 packages built:</summary>
  <ul>
    <li>alcom</li>
    <li>arnis</li>
    <li>atuin-desktop</li>
    <li>balatro-mod-manager</li>
    <li>cargo-tauri</li>
    <li>catppuccinifier-gui</li>
    <li>chiri</li>
    <li>cinny-desktop</li>
    <li>clash-verge-rev</li>
    <li>devpod-desktop</li>
    <li>dorion</li>
    <li>en-croissant</li>
    <li>fedistar</li>
    <li>firezone-gui-client</li>
    <li>flying-carpet</li>
    <li>gale</li>
    <li>gg-jj</li>
    <li>headphones-toolbox</li>
    <li>insulator2</li>
    <li>kide</li>
    <li>lrcget</li>
    <li>modrinth-app</li>
    <li>modrinth-app-unwrapped</li>
    <li>motrix-next</li>
    <li>neohtop</li>
    <li>noriskclient-launcher</li>
    <li>noriskclient-launcher-unwrapped</li>
    <li>opencode-desktop</li>
    <li>overlayed</li>
    <li>pawn-appetit</li>
    <li>qbz</li>
    <li>quantframe</li>
    <li>radicle-desktop</li>
    <li>rapidraw</li>
    <li>rclone-ui</li>
    <li>readest</li>
    <li>restic-browser</li>
    <li>rquickshare</li>
    <li>samira</li>
    <li>slimevr</li>
    <li>sone</li>
    <li>stirling-pdf-desktop</li>
    <li>surrealist</li>
    <li>tabularis</li>
    <li>treedome</li>
    <li>trgui-ng</li>
    <li>wealthfolio</li>
    <li>yaak</li>
    <li>zmk-studio</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>cargo-tauri_1</summary>
<pre>Running phase: unpackPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/xf2b6qhljjw5470wcmkwbv3gmxpgb3z1-source
source root is source/tooling/cli
Executing cargoSetupPostUnpackHook
Finished cargoSetupPostUnpackHook
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
substitute(): ERROR: file &#x27;crates/tauri/Cargo.toml&#x27; does not exist</pre>
</details>
<details>
<summary>sftool-gui</summary>
<pre>node_modules/.pnpm/update-browserslist-db@1.1.3_browserslist@4.24.5/node_modules/update-browserslist-db/node_modules/.bin/browserslist: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/i27rhb3nr65rkrwz36bchkwmav6ggsmn-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/postcss-load-config@4.0.2_postcss@8.5.3/node_modules/postcss-load-config/node_modules/.bin/yaml: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/i27rhb3nr65rkrwz36bchkwmav6ggsmn-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/vue-demi@0.14.10_vue@3.5.13_typescript@5.6.3_/node_modules/vue-demi/bin/vue-demi-switch.js: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/785jidgnryzj566s25s3rb262d4g5znb-nodejs-24.14.1/bin/node&quot;
node_modules/.pnpm/vue-demi@0.14.10_vue@3.5.13_typescript@5.6.3_/node_modules/vue-demi/bin/vue-demi-fix.js: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/785jidgnryzj566s25s3rb262d4g5znb-nodejs-24.14.1/bin/node&quot;
node_modules/.pnpm/sucrase@3.35.0/node_modules/sucrase/bin/sucrase-node: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/785jidgnryzj566s25s3rb262d4g5znb-nodejs-24.14.1/bin/node&quot;
node_modules/.pnpm/sucrase@3.35.0/node_modules/sucrase/bin/sucrase: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/785jidgnryzj566s25s3rb262d4g5znb-nodejs-24.14.1/bin/node&quot;
node_modules/.pnpm/sucrase@3.35.0/node_modules/sucrase/node_modules/.bin/glob: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/i27rhb3nr65rkrwz36bchkwmav6ggsmn-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/nanoid@3.3.11/node_modules/nanoid/bin/nanoid.cjs: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/785jidgnryzj566s25s3rb262d4g5znb-nodejs-24.14.1/bin/node&quot;
node_modules/.pnpm/cross-spawn@7.0.6/node_modules/cross-spawn/node_modules/.bin/node-which: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/i27rhb3nr65rkrwz36bchkwmav6ggsmn-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/css-selector-tokenizer@0.8.0/node_modules/css-selector-tokenizer/node_modules/.bin/cssesc: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/i27rhb3nr65rkrwz36bchkwmav6ggsmn-bash-5.3p9/bin/sh&quot;
Finished pnpmConfigHook
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Executing tauriBuildHook
/build/source/src-tauri /build/source
cargo-tauri.hook cargoFlags: -j 12 --target x86_64-unknown-linux-gnu --offline --profile release
cargo-tauri.hook tauriFlags: --bundles deb --target x86_64-unknown-linux-gnu
        Info Looking up installed tauri packages to check mismatched versions...
       Error Found version mismatched Tauri packages. Make sure the NPM package and Rust crate versions are on the same major/minor releases:
tauri (v2.6.2) : @tauri-apps/api (v2.5.0)</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 11 packages failed to build:</summary>
  <ul>
    <li>cargo-tauri_1</li>
    <li>chiri</li>
    <li>cinny-desktop</li>
    <li>en-croissant</li>
    <li>modrinth-app</li>
    <li>modrinth-app-unwrapped</li>
    <li>pawn-appetit</li>
    <li>sftool-gui</li>
    <li>stirling-pdf-desktop</li>
    <li>trgui-ng</li>
    <li>venator</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 21 packages built:</summary>
  <ul>
    <li>arnis</li>
    <li>atuin-desktop</li>
    <li>cargo-tauri</li>
    <li>devpod-desktop</li>
    <li>dorion</li>
    <li>fedistar</li>
    <li>gg-jj</li>
    <li>gitbutler</li>
    <li>kanri</li>
    <li>kide</li>
    <li>lrcget</li>
    <li>motrix-next</li>
    <li>neohtop</li>
    <li>opencode-desktop</li>
    <li>radicle-desktop</li>
    <li>rapidraw</li>
    <li>restic-browser</li>
    <li>rquickshare</li>
    <li>tabularis</li>
    <li>yaak</li>
    <li>zmk-studio</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>cargo-tauri_1</summary>
<pre>Running phase: unpackPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/xf2b6qhljjw5470wcmkwbv3gmxpgb3z1-source
source root is source/tooling/cli
Executing cargoSetupPostUnpackHook</pre>
</details>
<details>
<summary>chiri</summary>
<pre>node_modules/.pnpm/vite@8.0.3_@types+node@25.5.2_esbuild@0.27.2_jiti@1.21.7/node_modules/vite/node_modules/.bin/esbuild: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/marked@17.0.5/node_modules/marked/bin/marked.js: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/0zb2cr3b9xlancr6s938s73pxfcbzji2-nodejs-22.22.2/bin/node&quot;
Finished pnpmConfigHook
configurePhase completed in 1 minutes 15 seconds
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Executing tauriBuildHook

&gt; chiri@0.8.0 build /nix/build/nix-51590-2169678592/source
&gt; tsc &amp;&amp; vite build

error during build:
Error: getaddrinfo ENOTFOUND localhost
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:internal/dns/promises:88:17) {
  errno: -3008,
  code: &#x27;ENOTFOUND&#x27;,
  syscall: &#x27;getaddrinfo&#x27;,
  hostname: &#x27;localhost&#x27;
}
 ELIFECYCLE  Command failed with exit code 1.</pre>
</details>
<details>
<summary>cinny-desktop</summary>
<pre>  cargo:warning=Inherited flag &quot;-fno-omit-frame-pointer&quot; is not supported by the currently used CC
  cargo:warning=Warning: supplying the --target arm64-apple-macosx != arm64-apple-darwin argument to a nix-wrapped compiler may not work correctly - cc-wrapper is currently not designed with multi-target compilers in mind. You may want to use an un-wrapped compiler instead.
  cargo:warning=error: unable to open output file &#x27;/homeless-shelter/.cache/clang/ModuleCache/2J108BHMUNSDR/Cocoa-3H6ZQ4R9G9QAO.pcm&#x27;: &#x27;No such file or directory&#x27;
  cargo:warning=In file included from objc/notify.m:1:
  cargo:warning=objc/notify.h:1:9: fatal error: could not build module &#x27;Cocoa&#x27;
  cargo:warning=    1 | #import &lt;Cocoa/Cocoa.h&gt;
  cargo:warning=      |  ~~~~~~~^
  cargo:warning=error: unable to open output file &#x27;/homeless-shelter/.cache/clang/ModuleCache/2J108BHMUNSDR/CoreServices-3BJQO8Q15IGO6.pcm&#x27;: &#x27;No such file or directory&#x27;
  cargo:warning=error: unable to open output file &#x27;/homeless-shelter/.cache/clang/ModuleCache/2J108BHMUNSDR/Foundation-3HXFEQF3GN24N.pcm&#x27;: &#x27;No such file or directory&#x27;
  cargo:warning=error: unable to open output file &#x27;/homeless-shelter/.cache/clang/ModuleCache/2J108BHMUNSDR/ObjectiveC-1QU4Z5UK6R2G2.pcm&#x27;: &#x27;No such file or directory&#x27;
  cargo:warning=5 errors generated.

  --- stderr


  error occurred in cc-rs: command did not execute successfully (status code exit status: 1): env -u IPHONEOS_DEPLOYMENT_TARGET LC_ALL=&quot;C&quot; &quot;/nix/store/rwsbw420hcq148hifv7fga6r4g0k1dw8-clang-wrapper-21.1.8/bin/cc&quot; &quot;-O3&quot; &quot;-ffunction-sections&quot; &quot;-fdata-sections&quot; &quot;-fPIC&quot; &quot;--target=arm64-apple-macosx&quot; &quot;-mmacosx-version-min=10.13&quot; &quot;-Wall&quot; &quot;-Wextra&quot; &quot;-fmodules&quot; &quot;-Wno-deprecated-declarations&quot; &quot;-mmacos-version-min=10.13&quot; &quot;-o&quot; &quot;/nix/build/nix-51590-2169678593/source/src-tauri/target/aarch64-apple-darwin/release/build/mac-notification-sys-1450b20cdc7d4dee/out/54532c69c7c8f551-notify.o&quot; &quot;-c&quot; &quot;objc/notify.m&quot;


warning: build failed, waiting for other jobs to finish...
       Error failed to build app: failed to build app</pre>
</details>
<details>
<summary>en-croissant</summary>
<pre>cargo-tauri.hook cargoFlags: -j 3 --target aarch64-apple-darwin --offline --profile release
cargo-tauri.hook tauriFlags: --bundles app --target aarch64-apple-darwin
        Info Looking up installed tauri packages to check mismatched versions...
        Warn The bundle identifier &quot;org.encroissant.app&quot; set in `&quot;tauri.conf.json&quot; identifier` ends with `.app`. This is not recommended because it conflicts with the application bundle extension on macOS.
     Running beforeBuildCommand `pnpm build-vite`
        Warn sw_vers command failed with Os { code: 2, kind: NotFound, message: &quot;No such file or directory&quot; }

&gt; en-croissant@0.15.0 build-vite /nix/build/nix-51590-2169678596/source
&gt; tsgo --noEmit &amp;&amp; vite build

error during build:
Error: getaddrinfo ENOTFOUND localhost
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:internal/dns/promises:88:17) {
  errno: -3008,
  code: &#x27;ENOTFOUND&#x27;,
  syscall: &#x27;getaddrinfo&#x27;,
  hostname: &#x27;localhost&#x27;
}
 ELIFECYCLE  Command failed with exit code 1.
       Error beforeBuildCommand `pnpm build-vite` failed with exit code 1</pre>
</details>
<details>
<summary>modrinth-app-unwrapped</summary>
<pre>  2026-05-01T12:59:56.251+0000 [INFO] [org.gradle.launcher.daemon.server.DaemonRegistryUpdater] Advertising the daemon address to the clients: [042a6425-8d9f-4245-a069-7d8a2ef70393 port:49473, addresses:[localhost/127.0.0.1]]
  2026-05-01T12:59:56.252+0000 [DEBUG] [org.gradle.launcher.daemon.server.DaemonRegistryUpdater] Advertised daemon context: DefaultDaemonContext[uid=b6535899-89c9-4e13-a277-b72768d1f7eb,javaHome=/nix/store/f3m44k6jy7if29fgpkskislcw0zawi1d-zulu-ca-jdk-17.0.18/Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home,javaVersion=17,javaVendor=Azul Systems, Inc.,daemonRegistryDir=/nix/build/nix-51590-2169678603/tmp.Xf1twBLR9K/daemon,pid=71619,idleTimeout=120000,priority=NORMAL,applyInstrumentationAgent=true,nativeServicesMode=ENABLED,daemonOpts=-XX:MaxMetaspaceSize=384m,-XX:+HeapDumpOnOutOfMemoryError,-Xms256m,-Xmx512m,-Dfile.encoding=utf-8,-Djavax.net.ssl.trustStore=/nix/build/nix-51590-2169678603/tmp.dG4e6jBsZE/keystore,-Djavax.net.ssl.trustStorePassword=ZYCYDDQ5AP6JL4MR,-Duser.country=US,-Duser.language=en,-Duser.variant]
  2026-05-01T12:59:56.252+0000 [DEBUG] [org.gradle.launcher.daemon.registry.PersistentDaemonRegistry] Storing daemon address: [042a6425-8d9f-4245-a069-7d8a2ef70393 port:49473, addresses:[localhost/127.0.0.1]], context: DefaultDaemonContext[uid=b6535899-89c9-4e13-a277-b72768d1f7eb,javaHome=/nix/store/f3m44k6jy7if29fgpkskislcw0zawi1d-zulu-ca-jdk-17.0.18/Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home,javaVersion=17,javaVendor=Azul Systems, Inc.,daemonRegistryDir=/nix/build/nix-51590-2169678603/tmp.Xf1twBLR9K/daemon,pid=71619,idleTimeout=120000,priority=NORMAL,applyInstrumentationAgent=true,nativeServicesMode=ENABLED,daemonOpts=-XX:MaxMetaspaceSize=384m,-XX:+HeapDumpOnOutOfMemoryError,-Xms256m,-Xmx512m,-Dfile.encoding=utf-8,-Djavax.net.ssl.trustStore=/nix/build/nix-51590-2169678603/tmp.dG4e6jBsZE/keystore,-Djavax.net.ssl.trustStorePassword=ZYCYDDQ5AP6JL4MR,-Duser.country=US,-Duser.language=en,-Duser.variant]
  2026-05-01T12:59:56.257+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire exclusive lock on daemon addresses registry.
  2026-05-01T12:59:56.300+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
  2026-05-01T12:59:56.301+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
  2026-05-01T12:59:56.303+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire exclusive lock on daemon addresses registry.
  2026-05-01T12:59:56.303+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
  2026-05-01T12:59:56.307+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
  2026-05-01T12:59:56.307+0000 [LIFECYCLE] [org.gradle.launcher.daemon.server.Daemon] Daemon server started.
  2026-05-01T12:59:56.308+0000 [DEBUG] [org.gradle.launcher.daemon.bootstrap.DaemonStartupCommunication] Completed writing the daemon greeting. Closing streams...----- End of the daemon log -----


  * Try:
  &gt; Run with --stacktrace option to get the stack trace.
  &gt; Run with --info or --debug option to get more log output.
  &gt; Run with --scan to get full insights from a Build Scan (powered by Develocity).
  &gt; Get more help at https://help.gradle.org./
warning: build failed, waiting for other jobs to finish...
       Error failed to build app: failed to build app</pre>
</details>
<details>
<summary>pawn-appetit</summary>
<pre>node_modules/.pnpm/tsx@4.20.6/node_modules/tsx/dist/cli.mjs: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node&quot;
node_modules/.pnpm/tsx@4.20.6/node_modules/tsx/node_modules/.bin/esbuild: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/prop-types@15.8.1/node_modules/prop-types/node_modules/.bin/loose-envify: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/esbuild@0.25.11/node_modules/esbuild/bin/esbuild: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node&quot;
node_modules/.pnpm/markdown-it@14.1.0/node_modules/markdown-it/bin/markdown-it.mjs: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node&quot;
Finished pnpmConfigHook
configurePhase completed in 1 minutes 7 seconds
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Executing tauriBuildHook
/nix/build/nix-51590-2169678607/source/src-tauri /nix/build/nix-51590-2169678607/source
cargo-tauri.hook cargoFlags: -j 3 --target aarch64-apple-darwin --offline --profile release
cargo-tauri.hook tauriFlags: --bundles app --target aarch64-apple-darwin
        Info Looking up installed tauri packages to check mismatched versions...
     Running beforeBuildCommand `pnpm build-vite`
        Warn sw_vers command failed with Os { code: 2, kind: NotFound, message: &quot;No such file or directory&quot; }

&gt; pawn-appetit@0.11.0 build-vite /nix/build/nix-51590-2169678607/source
&gt; tsc --noEmit &amp;&amp; vite build
</pre>
</details>
<details>
<summary>sftool-gui</summary>
<pre>node_modules/.pnpm/vite@6.3.4_jiti@2.4.2_lightningcss@1.29.2_yaml@2.7.1/node_modules/vite/node_modules/.bin/yaml: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/vite@6.3.4_jiti@2.4.2_lightningcss@1.29.2_yaml@2.7.1/node_modules/vite/node_modules/.bin/esbuild: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/update-browserslist-db@1.1.3_browserslist@4.24.5/node_modules/update-browserslist-db/node_modules/.bin/browserslist: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/update-browserslist-db@1.1.3_browserslist@4.24.5/node_modules/update-browserslist-db/cli.js: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node&quot;
node_modules/.pnpm/autoprefixer@10.4.21_postcss@8.5.3/node_modules/autoprefixer/bin/autoprefixer: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node&quot;
node_modules/.pnpm/autoprefixer@10.4.21_postcss@8.5.3/node_modules/autoprefixer/node_modules/.bin/browserslist: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/postcss@8.5.3/node_modules/postcss/node_modules/.bin/nanoid: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/@vue+compiler-vue2@2.7.16/node_modules/@vue/compiler-vue2/node_modules/.bin/he: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/@vue+compiler-core@3.5.13/node_modules/@vue/compiler-core/node_modules/.bin/parser: interpreter directive changed from &quot;#!/bin/sh&quot; to &quot;/nix/store/f700nj7wlwg441h39gkq29qbviy99sgq-bash-5.3p9/bin/sh&quot;
node_modules/.pnpm/yaml@2.7.1/node_modules/yaml/bin.mjs: interpreter directive changed from &quot;#!/usr/bin/env node&quot; to &quot;/nix/store/sf85v1qd1glkmsyfxwn9p8wridb4qf6y-nodejs-24.14.1/bin/node&quot;
Finished pnpmConfigHook
Running phase: buildPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;buildPhase&quot; }
Executing tauriBuildHook
/nix/build/nix-51590-2169678612/source/src-tauri /nix/build/nix-51590-2169678612/source
cargo-tauri.hook cargoFlags: -j 3 --target aarch64-apple-darwin --offline --profile release
cargo-tauri.hook tauriFlags: --bundles app --target aarch64-apple-darwin
        Info Looking up installed tauri packages to check mismatched versions...
       Error Found version mismatched Tauri packages. Make sure the NPM package and Rust crate versions are on the same major/minor releases:
tauri (v2.6.2) : @tauri-apps/api (v2.5.0)</pre>
</details>
<details>
<summary>stirling-pdf-desktop</summary>
<pre>2026-05-01T14:02:42.095+0000 [DEBUG] [org.gradle.launcher.daemon.server.DaemonRegistryUpdater] Advertised daemon context: DefaultDaemonContext[uid=f0c5730c-a730-4522-81c1-d210202339a1,javaHome=/nix/store/dswnyh8p966zx432y7kqby2v7i0pahqc-zulu-ca-jdk-25.0.0/Library/Java/JavaVirtualMachines/zulu-25.jdk/Contents/Home,javaVersion=25,javaVendor=Azul Systems, Inc.,daemonRegistryDir=/nix/build/nix-51590-2169678613/tmp.XyjyjukRNU/daemon,pid=82416,idleTimeout=120000,priority=NORMAL,applyInstrumentationAgent=true,nativeServicesMode=ENABLED,daemonOpts=-XX:MaxMetaspaceSize=384m,-XX:+HeapDumpOnOutOfMemoryError,-Xms256m,-Xmx512m,-Dfile.encoding=UTF-8,-Djavax.net.ssl.trustStore=/nix/build/nix-51590-2169678613/tmp.6ej7rDkQlj/keystore,-Djavax.net.ssl.trustStorePassword=GVRGUUUEXYFA6U4O,-Duser.country=US,-Duser.language=en,-Duser.variant]
2026-05-01T14:02:42.096+0000 [DEBUG] [org.gradle.launcher.daemon.registry.PersistentDaemonRegistry] Storing daemon address: [b91c6c04-f41f-4cbb-9918-7816c38f88a9 port:49487, addresses:[localhost/127.0.0.1]], context: DefaultDaemonContext[uid=f0c5730c-a730-4522-81c1-d210202339a1,javaHome=/nix/store/dswnyh8p966zx432y7kqby2v7i0pahqc-zulu-ca-jdk-25.0.0/Library/Java/JavaVirtualMachines/zulu-25.jdk/Contents/Home,javaVersion=25,javaVendor=Azul Systems, Inc.,daemonRegistryDir=/nix/build/nix-51590-2169678613/tmp.XyjyjukRNU/daemon,pid=82416,idleTimeout=120000,priority=NORMAL,applyInstrumentationAgent=true,nativeServicesMode=ENABLED,daemonOpts=-XX:MaxMetaspaceSize=384m,-XX:+HeapDumpOnOutOfMemoryError,-Xms256m,-Xmx512m,-Dfile.encoding=UTF-8,-Djavax.net.ssl.trustStore=/nix/build/nix-51590-2169678613/tmp.6ej7rDkQlj/keystore,-Djavax.net.ssl.trustStorePassword=GVRGUUUEXYFA6U4O,-Duser.country=US,-Duser.language=en,-Duser.variant]
2026-05-01T14:02:42.101+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire exclusive lock on daemon addresses registry.
2026-05-01T14:02:42.103+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2026-05-01T14:02:42.104+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
2026-05-01T14:02:42.105+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire exclusive lock on daemon addresses registry.
2026-05-01T14:02:42.105+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
2026-05-01T14:02:42.107+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
2026-05-01T14:02:42.107+0000 [LIFECYCLE] [org.gradle.launcher.daemon.server.Daemon] Daemon server started.
2026-05-01T14:02:42.108+0000 [DEBUG] [org.gradle.launcher.daemon.bootstrap.DaemonStartupCommunication] Completed writing the daemon greeting. Closing streams...
2026-05-01T14:02:42.114+0000 [DEBUG] [org.gradle.launcher.daemon.server.Daemon] stopOnExpiration() called on daemon
2026-05-01T14:02:42.114+0000 [DEBUG] [org.gradle.launcher.daemon.server.Daemon] awaitExpiration() called on daemon
2026-05-01T14:02:42.114+0000 [DEBUG] [org.gradle.launcher.daemon.server.DaemonStateCoordinator] daemon is running. Sleeping until state changes.----- End of the daemon log -----


* Try:
&gt; Run with --stacktrace option to get the stack trace.
&gt; Run with --info or --debug option to get more log output.
&gt; Run with --scan to get full insights.
&gt; Get more help at https://help.gradle.org./</pre>
</details>
<details>
<summary>trgui-ng</summary>
<pre>   Compiling security-framework v2.11.1
   Compiling pxfm v0.1.25
   Compiling tempfile v3.23.0
error: failed to run custom build command for `coreaudio-sys v0.2.17`

Caused by:
  process didn&#x27;t exit successfully: `/nix/build/nix-51590-2169678589/source/target/release/build/coreaudio-sys-2b083b8e266d43b8/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=COREAUDIO_SDK_PATH
  cargo:rustc-link-lib=framework=AudioUnit
  cargo:rustc-link-lib=framework=CoreAudio
  cargo:rerun-if-env-changed=BINDGEN_EXTRA_CLANG_ARGS

  --- stderr

  thread &#x27;main&#x27; (551454) panicked at /nix/build/nix-51590-2169678589/TrguiNG-1.5.1-vendor/source-registry-0/bindgen-0.72.1/lib.rs:616:27:
  Unable to find libclang: &quot;couldn&#x27;t find any valid shared libraries matching: [&#x27;libclang.dylib&#x27;], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])&quot;
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
       Error failed to build app: failed to build app</pre>
</details>
</details>

--- a/pkgs/by-name/ca/cargo-tauri/package.nix
+++ b/pkgs/by-name/ca/cargo-tauri/package.nix
@@ -14,20 +14,26 @@
 
 rustPlatform.buildRustPackage (finalAttrs: {
   pname = "tauri";
-  version = "2.9.6";
+  version = "2.11.0";
 
   src = fetchFromGitHub {
     owner = "tauri-apps";
     repo = "tauri";
     tag = "tauri-cli-v${finalAttrs.version}";
-    hash = "sha256-VtZxFkxOLMNwl3A/2qoNJ/HXr5FXFKQYw+ri5Yp8eOE=";
+    hash = "sha256-hbTpLeWMc5nSJeq1vWKbVpfhp1KZjiBgmaiQMUs/MIQ=";
   };
 
-  cargoHash = "sha256-uAjEQBHDpVv73MbeoU86tObiXSUKKjImpMTLHXKMRNs=";
+  cargoHash = "sha256-PrvlPfK/9qyBWOs6hG3C3w9TMK7p1TCoKpjSWkhN2rY=";
 
   nativeBuildInputs = lib.optionals (stdenv.hostPlatform.isDarwin || stdenv.hostPlatform.isLinux) [
     pkg-config
   ];
+
+  # Explicitly enable optional `rustls` dependency.
+  postPatch = ''
+    substituteInPlace crates/tauri/Cargo.toml \
+      --replace-fail 'dep:rustls' 'rustls'
+  '';
 
   buildInputs =
     # Required for tauri-macos-sign and RPM support in tauri-bundler

--- a/pkgs/by-name/ca/cargo-tauri/test-app.nix
+++ b/pkgs/by-name/ca/cargo-tauri/test-app.nix
@@ -34,7 +34,7 @@ stdenv.mkDerivation (finalAttrs: {
     pnpm = pnpm_9;
 
     fetcherVersion = 3;
-    hash = "sha256-/g+2jZQq3nTJnKpj0PlT6zB3UcUBE2ND8797XRwVZ0s=";
+    hash = "sha256-DiRho0tmR/4XJi7cAZxesVA1iR8uiwknCfmIP2x9i5I=";
   };
 
   nativeBuildInputs = [

--- a/pkgs/by-name/ca/cargo-tauri_1/package.nix
+++ b/pkgs/by-name/ca/cargo-tauri_1/package.nix
@@ -19,6 +19,8 @@ cargo-tauri.overrideAttrs (
 
     patches = [ ];
 
+    postPatch = "";
+
     # Manually specify the sourceRoot since this crate depends on other crates in the workspace. Relevant info at
     # https://discourse.nixos.org/t/difficulty-using-buildrustpackage-with-a-src-containing-multiple-cargo-workspaces/10202
     sourceRoot = "${finalAttrs.src.name}/tooling/cli";

